website/docs/exhaustiveness.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: exhaustiveness
-title: Exhaustiveness Checking
+title: Exhaustiveness (T.absurd)
```

Comment:
These extra little bits to express the labels in terms of user concepts instead of internals concepts are great.